decentraland/unity-renderer
# feat: add new design for place user count

## What does this PR change?

Fix #4531 
Adds the new UI design to always show the user count for places in menu

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?explorer-branch=feat/places-user-count
2. Open the explore
3. Verify that the places cards work as expected
4. Verify that the user count is always visible in the top left

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
